common/src/main/resources/WindowsMemoryInspector.ps1

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+param (
```

Comment:
@zhou9584 Will the new added script be packaged automatically? Otherwise the memory inspector will not run as expected if we lost this script

Reply:
Yes, the file will be packaged, and we just need to copy it from jar to outside before executing.

Reply:
Do we need to add any clarify in the code base to do the expression?

Reply:
How about adding a README.md under `common\src\main\resources\`. Before we do that, maybe we need to make sure the README.md will not be packaged. Do you know if the README.md will be packaged? cc @dinghong2233 @zhou9584